Create accessible database localhost

jforster19 opened the pull request:
Loaded scripts for database reading/writing of Github repo input files

jforster19 committed:
Push valid database reference utilizing MySQL database

Loading folder for RMD file to be able to access data in database based on schema prepared in the initial stage of Project 3. The MySQL database is accessed as a local host and user must have that configured correctly (SET GLOBAL local_infile=1; show global variables like 'local_infile';) on localhost to confirm write privileges are enabled.

jforster19 committed:
Loaded RMD file to write/read data

Allows users to be able to create mysql database based on setup on their local machine. RMD file prompts for key credentials in accessing database (username, password,schema,host). Please confirm write privledges on MySQL to be able to properly load data into created schema.

Anthogonyst reviewed (changes requested):
Please delete read_me.txt and move *.Rmd to vignettes. Additionally, please fix the filepaths in load_file_paths.

jforster19 committed:
Delete Database directory

move to vignette location

jforster19 committed:
Load GitHub repo data into MySQL database

Allows users to be able to create mysql database based on setup on their local machine. RMD file prompts for key credentials in accessing database (username, password,schema,host). Please confirm write privledges on MySQL to be able to properly load data into created schema.

Anthogonyst reviewed (changes requested):
Please add the file extension.

jforster19 committed:
Rename load_db_into_db to load_db_into_db.rmd

Anthogonyst reviewed (changes requested):
The correct extension is Rmd with a capital R.

jforster19 committed:
Rename load_db_into_db.rmd to load_db_into_db.Rmd
